spec/services/content_validator_spec.rb

```diff
@@ -57,7 +57,9 @@
 
   describe 'custom validation' do
     it 'raises issue if the summary is not set' do
-      document = build(:document, :with_body_in_schema, contents: { body: "Too short" })
+      body_field = { id: "body", label: "Body", type: "govspeak", validations: { "min_length" => 10 } }
```

Comment:
The number of times we have a field hash is making me think we should have a Field factory and use attributes_for - what do you think?